src/components/Button/Button.css

```
@@ -0,0 +1,91 @@
+/*
+* Stylesheet for buttons
+* @author: Dory A.Azar
+*/
+
+.default-button {
+
+  background-color: #f8f9fa;
```

You reference the same color a lot, so pull it out into a css variable (or constant in our case). 
```suggestion
  background-color: var(--primary-color);
```